apps/voice-notifies/voice-notifies.yml

```diff
@@ -0,0 +1,15 @@
+name: 'Voice Notifies'
+description: 'Google Voice compatible desktop app'
```

Comment:
This should be the way you had it one commit ago. 

I.e. `XXXX compatible with Google Voice`